### DISCUSSION TITLE
Hint markers color changed after updating Firefox to version 95.0

### DISCUSSION THREAD

**Discussion by vort3** [gdh1995/vimium-c#517] [Q&A] [answered]
I can't tell exactly what changed, but I guess they are now darker?
Firefox auto updated a few minutes ago and I immediately noticed that something is wrong with them, they are not as nice to llok at as before.

Win 10x64, Firefox with a dark theme, Windows theme is Light.

**Comment by vort3** [comment]
Here's a screenshot of how they look on my screen:
![image](https://user-images.githubusercontent.com/12686404/145672063-32820a10-a350-4560-9bf5-e21e0cd2662f.png)

**Comment by gdh1995** [comment, marked as answer]
You might turn off the advanced option named "auto dark theme".



---Original---
From: ***@***.***&gt;
Date: Sat, Dec 11, 2021 17:44 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [gdh1995/vimium-c] Hint markers color changed after updatingFirefox to version 95.0 (Discussion #517)




 
Here's a screenshot of how they look on my screen:
 
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by vort3** [reply]
That's it, now my hint markers are brighter, thanks!
Why did that happen after updating firefox? Does new (v. 95) firefox have some dark theme changes?

**Comment by gdh1995** [comment]
By default, Vimium C has enabled its `Auto switch between light and dark mode`, and will ask Firefox for the status of `prefers-color-scheme` to decide whether to make its UI dark or not.

Since version 67, Firefox will tell the status of system-level dark mode to Vimium C, but nothing about its browser-level theme. And Firefox 95 improves its built-in "Dark" theme and returns `dark` to Vimium C if you've enabled the "Dark" theme.
